test/unit/specs/components/ActivityCollector/attachClickActivityCollector.spec.js

```diff
@@ -88,6 +89,51 @@ describe("ActivityCollector::attachClickActivityCollector", () => {
     });
   });
 
+  it("Sends an event if onBeforeLinkCollect does not return false", () => {
+    eventManager.createEvent = () => {
+      return {
+        isEmpty: () => false
+      };
+    };
+    spyOn(eventManager, "sendEvent").and.callThrough();
+    config.onBeforeLinkCollect = function() {};
+    spyOn(config, "onBeforeLinkCollect").and.callThrough();
```

Comment:
The easiest way to do these two lines is like this: 

`config.onBeforeLinkCollect = jasmine.createSpy();`